.hlint.yaml

```diff
@@ -54,6 +54,7 @@
 - ignore: {name: "Redundant multi-way if"} # Using this style might be appropriate given the circumstances.
 - ignore: {name: "Use const"} # Sometimes using a lambda expression is simpler and clearer.
 - ignore: {name: "Redundant id"} # Sometimes this is useful for vertical alignment of elements.
+- ignore: {name: "Use camelCase"} # Sometimes this is useful in test code.
```

Comment:
I think we normally want this hlint enabled for application code.

Reply:
Fair enough!

I've moved the suppression of this warning to just the `RoundRobinSpec` module.

Fixed in: b9454787a8184e0e83338602147614f56763c5bd